### PR TITLE
Recommendation description bug

### PR DESCRIPTION
**Pull request by kristinh12**
Issue: https://github.com/lytics/pathforajs/issues/298

Previously if a recommendation was missing a description, the modal wouldn't display.  

I added to the conditional to check for html tags before trying to set the color, and wrote a test.

Modal successfully displays:
screen shot 2018-06-27 at 2 11 19 pm

### CODE REVIEW

File: test.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
```
Comment (prosdev) [inline test.html]: @kristinh12 Can we be a bit more descriptive with this file name? `test.html` doesn't give that much context. Maybe `widget-no-description-test.html` or something like that.

Comment (kristinh12) [reply test.html]: Ah I can probably just delete that, I wrote an actual test widget.spec.js.  The test.html was to try to duplicate the problem and have a visual of it working.

Comment (prosdev) [reply test.html]: I think these are super help though. Maybe we can add a folder to keep all of these? What do you think @sarahyablok ?

Comment (sarahyablok) [reply test.html]: We had an examples folder which got deleted cause it had bad examples. Perhaps for now we can delete this and file a ticket to create a new ticket for a new examples folder.

File: dist/pathfora.js [hunk 2]
```diff
@@ -2581,11 +2581,11 @@ function setCustomColors (widget, colors) {
       contentUnit.style.setProperty('background-color', colors.actionBackground, 'important');
     }
 
-    if (colors.actionText) {
+    if (colors.actionText && contentUnitMeta.querySelector('h4')) {
```
Comment (sarahyablok) [inline dist/pathfora.js]: This is a tiny efficiency nit, but you can actually define this as a variable up in the jumble of vars at the top of this function, that saves us from doing the querySelector lookup twice.

```
var contentUnitMetaTitle = contentUnitMeta.querySelector('h4');

...

if (colors.actionText && contentUnitMetaTitle) {
  contentUnitMetaTitle.style.setProperty('color', colors.actionText, 'important');
}
```

File: test.html (hunk 1 above)
Comment (prosdev) [reply test.html]: Sounds good! I'm ok with that! 👍

File: test/widget.spec.js [hunk 3]
```diff
@@ -492,6 +492,38 @@ describe('Widgets', function () {
...
+        title: 'test',
+        default: true
+      }]
+    });
+    window.pathfora.initializeWidgets([modal]);
+
+    var widget = $('#' + modal.id);
+    expect(widget).toBeDefined();
+  });
+
```
Comment (sarahyablok) [inline test/widget.spec.js]: I believe there is a `.spec` file for recommendation widgets, perhaps this test is better suited there.

**Pull request merged**